src/components/MarketingMaterials.module.scss

```diff
@@ -0,0 +1,78 @@
+
+.downloadContainer {
```

Comment:
Why are you adding Sass, I thought we were using tailwind. You should add the tailwind classes directly in the component.